config/paths.js

```diff
@@ -19,9 +19,7 @@ var isInNodeModules = (
 );
 
 // Are we developing create-react-app locally?
-var isInCreateReactAppSource = (
-  process.argv.some(arg => arg.indexOf('--debug-template') > -1)
-);
+var isInCreateReactAppSource = process.argv.indexOf('--debug-template') !== -1;
```

Comment:
We use `--smoke-test` in the e2e test, but I'm not 100% sure if that case can actually ever happen. (better safe than sorry though) @gaearon and @vjeux have a better idea of that part of the codebase!